canonical-websites/www.ubuntu.com
# Migrate global migration to Vanilla v1 in /about section

## Done

Migrate global migration to Vanilla v1 in /about section

## QA

- Pull code
- Run `./run serve`
- Go to file `templates/about/index.html`
- Change line 3 to `{% extends "about/base_about-v1.html" %}`
- Change line 8 to `{% include "templates/_nav_breadcrumb-v1.html" with section_title="About" page_title="The Ubuntu Project" %}`
- View [http://0.0.0.0:8001/about](http://0.0.0.0:8001/about) across breakpoints
- Check code

**Important notes:**


- This is a first pass, `styles-v1.scss` should be viewed as a WIP throughout Vanilla v1 migration
- This is a blocking PR so due to expediency, I have not yet split the different code overrides into separate partials. These are commented as @TODOs and will be moved out in due course.
- Site search in nav is currently hidden across all breakpoints, this will be completed as a separate issue

## Issue / Card

Fixes #1278
